src/EventStore.ClusterNode/Program.cs

```diff
@@ -30,7 +30,8 @@ public class Program : ProgramBase<ClusterNodeOptions>
 
         public static void Main(string[] args)
```

Comment:
The only thing that changed in this file is the formatting and is likely confusing for git history. Would you mind terrible reverting this entire file?